src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Runtimes/RegistryReaderNetCoreRuntimesVersions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.Win32;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Runtimes
+{
+    internal sealed class RegistryReaderNetCoreRuntimesVersions
```

Comment:
nit: Use adjectives before the noun of the class name. Also, the double plural is a bit confusing. Suggested: `NetCoreRuntimeVersionsRegistryReader`
Some other places in the repo with long names you'll see an `Of` convention too, which would be like `RegistryReaderOfNetCoreRuntimeVersions`, but ending a class name with an plural (`s`) is also confusing, so I don't suggest this.

Reply:
changed.